source/_integrations/tplink.markdown

```diff
@@ -40,7 +43,7 @@ There is currently support for the following device types within Home Assistant:
 
 See [Supported Devices in python-kasa](https://github.com/python-kasa/python-kasa#supported-devices) for an up to date list.
 
-## Supported devices
+## Supported devices not requiring authentication
```

Comment:
Maybe we should indent these to become below the "supported devices" section above? This way we could simplify these to become "No authentication required" and "Requires authentication" or something similar.

Reply:
Done